.travis.yml

```diff
@@ -9,11 +9,17 @@ env:
 services:
   - postgresql
   - redis-server
+  - docker
 install:
   - pip install -r requirements.txt
 script:
   - flake8
   - python manage.py test
+  - docker build -t pyslackers/website:$TRAVIS_BRANCH .
 matrix:
   allow_failures:
-    - "3.7-dev"
+    python: "3.7-dev"
```

Comment:
Since over time we will probably have multiple python version that we allow failures on, not just current+1, we should keep this as a list `- python: "3.7-dev"`